docs/production-install.rst

```diff
@@ -5,7 +5,7 @@
 Production Install
 ==================
 
-Currently Socorro is supported on RedHat Linux 6 (and clones like CentOS)
+Currently Socorro is supported on CentOS 7
```

Comment:
To be fair, this will likely work on any RHEL7-alike (i.e. RPM-based with systemd), including Scientific (etc) and possibly even Fedora.